README.md

```diff
@@ -31,6 +31,7 @@ file and it will be removed.
 - Ubuntu
 - CentOS 7
 - Amazon Linux
+- Microsoft Windows
```

Comment:
should we list the versions as I don't think we want to advertise that we support say windows 2003.